backbone/src/main/java/org/researchstack/backbone/ui/step/body/DurationQuestionBody.java

```diff
@@ -0,0 +1,129 @@
+package org.researchstack.backbone.ui.step.body;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.text.InputFilter;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.EditorInfo;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import org.researchstack.backbone.R;
+import org.researchstack.backbone.answerformat.DecimalAnswerFormat;
+import org.researchstack.backbone.answerformat.DurationAnswerFormat;
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.step.QuestionStep;
+import org.researchstack.backbone.step.Step;
+import org.researchstack.backbone.utils.TextUtils;
+import org.researchstack.backbone.utils.ViewUtils;
+
+/**
+ * Implementation of the question type "duration"
+ * the JSON can specify a unit and a step, here they are not used, and probably not either in ResearchKit
+ */
+public class DurationQuestionBody implements StepBody
+{
+    //-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
+    // Constructor Fields
+    //-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
+    private QuestionStep        step;
+    //I'm assuming the result is given as the total number of minutes
```

Comment:
I think you can change to simply say "The result should be given as the total number of minutes." and move it to the large comment block before the method declaration, above.